packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -856,15 +856,14 @@ export class LocalIntervalCollection<TInterval extends ISerializableInterval> {
         return newInterval;
     }
 
-    public serialize(): ISerializedIntervalCollectionV2 {
+    public serialize(snapshotIsNormalized: boolean): ISerializedIntervalCollectionV2 {
```

Comment:
Rather than change the public API of this method, you should be able to get a document in the same format as the old one by iterating over all of the intervals in your V1IntervalCollection. So the `.store` method could be something like

`return Array.from(this, (interval) => interval.serialize(this.client)) as unknown as ISerializedIntervalCollectionV2`.

That should get rid of the remaining public API changes, which is great!